src/components/modals/Settings.tsx

```diff
@@ -119,7 +119,16 @@ const ModalSettings = () => {
       title='Settings'
       actions={({ close }) => (
         <div className={css({ textAlign: 'center' })}>
-          <ActionButton key='close' title='Close' {...fastClick(() => close())} />
+          <ActionButton
+            key='close'
+            title='Close'
+            {...fastClick(() => {
+              close(true)
+              // if (Capacitor.isNativePlatform()) {
+              //   window.location.reload()
+              // }
```

Comment:
Is this comment unintentional, likely just used during testing? I believe it should be removed now.